src/test/scala/io/confluent/examples/streams/IntegrationTestScalaUtils.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.confluent.examples.streams
+
+import org.apache.kafka.common.serialization.{Deserializer, Serde, Serializer, Serdes => JSerdes}
+import org.apache.kafka.streams.kstream.{Windowed, WindowedSerdes}
+import org.apache.kafka.streams.{KeyValue, TopologyTestDriver}
+
+object IntegrationTestScalaUtils {
```

Comment:
Might be worth some comments explaining what's in here, and why it's needed.

Reply:
Ack